astropy/coordinates/builtin_frames/icrs_cirs_transforms.py

```diff
@@ -20,7 +20,42 @@
 from .gcrs import GCRS
 from .cirs import CIRS
 from .hcrs import HCRS
-from .utils import get_jd12, aticq, atciqz, get_cip, prepare_earth_position_vel
+from .utils import get_jd12, aticq, atciqz, get_cip
+
+
+# utility function for transforms
+def prepare_earth_position_vel(time):
+    """
+    Get barycentric position and velocity, and heliocentric position of Earth
+
+    Parameters
+    -----------
+    time : `~astropy.time.Time`
+        time at which to calculate position and velocity of Earth
+
+    Returns
+    --------
+    earth_pv : `np.ndarray`
+        Barycentric position and velocity of Earth, in au and au/day
+    earth_helio : `np.ndarray`
+        Heliocentric position of Earth in au
+    """
+    # this goes here to avoid circular import errors
+    from ..solar_system import (get_body_barycentric, get_body_barycentric_posvel,
+                                solar_system_ephemeris)
+    # get barycentric position and velocity of earth
+    ephemeris = solar_system_ephemeris.get()
```

Comment:
p.s. I guess not, since you have the inline import everywhere else. If you have time, maybe check whether you can put the import at the very end of the file, as that sometimes works.

Alternatively, maybe we're doing this the wrong way around? Most of `solar_system` does not actually need `ICRS`, `GCRS`, and `SkyCoord` -- perhaps it is best to move those imports locally instead? (to `get_body` and `_apparent_position...`).